inc/namespace.php

```diff
@@ -295,12 +325,12 @@ function get_elasticpress_indexable_post_types( array $types ) : array {
 /**
  * Override the elasticpress features should be enabled.
  *
- * @param boolean $is_active
- * @param array $settings
- * @param EP_Feature $feature
- * @return void
+ * @param boolean $is_active True if the feature is active.
+ * @param array $settings Feature settings array.
+ * @param EP_Feature $feature The feature object.
+ * @return bool
  */
-function override_elasticpress_feature_activation( bool $is_active, array $settings, EP_Feature $feature ) {
+function override_elasticpress_feature_activation( bool $is_active, array $settings, EP_Feature $feature ) : bool {
```

Comment:
If `$feature->slug` is `facets`, this won't be `bool` right?

Reply:
Ah I see. So the function should always return a boolean, we don't actually want to return the potential array value of the facets config in this instance. I updated the code to reflect this more consistently.